adhocracy4/reports/serializers.py

```diff
@@ -1,9 +1,24 @@
 from rest_framework import serializers
 
+from adhocracy4.api.dates import get_datetime_display
+
 from .models import Report
 
 
 class ReportSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(required=False, source="creator.username")
+    created_at = serializers.SerializerMethodField(required=False)
```

Comment:
maybe just use `created` here (and below) as we also use that in other places (e.g. comments)

Reply:
done